packages/fxa-auth-server/docs/metrics-events.md

```diff
@@ -390,10 +390,10 @@ has the same schema as `daily_multi_device_users`.
 | `device.created`       | A device record has been created for a Sync account.         |
 | `device.updated`       | Device record is updated on a Sync account.                  |
 | `device.deleted`       | Device record has been deleted from a Sync account.          |
-| `oauth.token.created`  | An oauth access token has been issued for an account. **      |
+| `oauth.token.created`  | An oauth access token has been issued for an account. \*\*   |
```

Comment:
Heh wow I'm really going to have to trigger another CI run for this because I forgot to address it with my last commit, [Prettier broke this section](https://github.com/mozilla/fxa/blob/23bb5e2cbf329909c73a4fb1fbc13c478dfa841d/packages/fxa-auth-server/docs/metrics-events.md#success-event-names) 😢